ar4_hardware_interface/urdf/ar4.urdf.xacro

```diff
@@ -23,4 +23,17 @@
     joint_offset_parameters_file="$(find ar4_hardware_interface)/config/joint_offsets/ar4.yaml"
   />
 
+  <xacro:if value="$(arg include_gripper)">
+    <!-- <xacro:include filename="$(find ar4_description)/urdf/ar_gripper_macro.xacro"/> -->
+    <xacro:include filename="$(find ar4_hardware_interface)/urdf/ar_gripper.ros2_control.xacro"/>
```

Comment:
We should change the name of this file `gripper.ros2_control.xacro` to match with the name used in Gazebo.

Reply:
Fixed.